fluid-cpp/fluidComputeSolution.cpp

```diff
@@ -1,8 +1,10 @@
-#include "FluidSolver.h"
+#include "FluidComputeSolution.h"
```

Comment:
You already changed the solvers and this was a very much needed work! :smiley:  Could you please go through the file diff and self-review the most important changes in the solvers? Add some comments on why some changes were needed (e.g. why removing the `_n` variables? What else should I pay attention at?): this will help you realize if there are any more bugs and it would help me review the PR.

Files I have in mind:
- [ ] `fluid-cpp/fluidComputeSolution.cpp`
- [ ] `fluid-cpp/fluidSolver.cpp` (especially since there is no diff after the renaming)
- [ ] `fluid-python/fluidSolver.py`
- [ ] `solid-cpp/solidComputeSolution.cpp`
- [ ] `solid-cpp/solidSolver.cpp`
- [ ] `solid-python/solidSolver.py`

Reply:
I looked over the solid and all the python solvers, I only need some help in reviewing the fluid C++ solver changes.